examples/python_scripting.py

```diff
@@ -29,12 +29,11 @@ def __init__(self):
         super().__init__(use_ipython=True)
         self._set_prompt()
         self.intro = 'Happy 𝛑 Day.  Note the full Unicode support:  😇 💩'
-        self.self_in_py = True
 
     def _set_prompt(self):
         """Set prompt so it displays the current working directory."""
         self.cwd = os.getcwd()
-        self.prompt = ansi.style('{!r} $ '.format(self.cwd), fg='cyan')
+        self.prompt = ansi.style(f'{self.cwd} $ ', fg='cyan')
```

Comment:
We should remove the f-strings so that the examples work with all currently supported versions of Python (3.5+).  

Other than that, this looks good.